### PR TITLE
Feature/argentina cc

### PR DESCRIPTION
**Pull request by andercampanha**
ON HOLD, waiting for #3 and #5 to be merged

### CODE REVIEW

File: woocommerce/lib/pages/checkout.js [hunk 1]
```diff
@@ -340,6 +340,9 @@ export default class Checkout {
   [choosePaymentMethod] (data) {
     const resolveMethod = (method, country) => {
       const elmMethods = {
+        ar: {
+          creditcard: 'credit-card-ar',
```
Comment (cristopher-rodrigues) [inline woocommerce/lib/pages/checkout.js]: What you think about this approach ? 🌊

File: woocommerce/lib/schemas/checkout.js [hunk 2]
```diff
@@ -39,6 +39,22 @@ export const CHECKOUT_SCHEMA = {
         )
       ).without('schema', [...R.keys(this.compliance()), ...['paymentType']]);
     },
+    creditcard() {
+      return Joi.object().keys(
+        Object.assign(
+          {},
+          this.compliance(),
+          {
+            schema: 'ArgentinaCreditCard',
```
Comment (cristopher-rodrigues) [inline woocommerce/lib/schemas/checkout.js]: What you think about we separate commons `schema objects` into a default to reuse, like in card, compliance required default data to all country, etc..

File: woocommerce/lib/pages/checkout.js (hunk 1 above)
Comment (andercampanha) [reply woocommerce/lib/pages/checkout.js]: I think it's the best way we have to reutilising defaults.js across all our projects and keep us from duplicating that file over and over again. It's very good.

File: woocommerce/lib/schemas/checkout.js (hunk 2 above)
Comment (andercampanha) [reply woocommerce/lib/schemas/checkout.js]: That's a great idea

**Pull request merged**